lib/blueprinter/base.rb

```diff
@@ -42,7 +43,8 @@ def self.prepare(object, view:)
       unless view_collection.has_view? view_name
         raise BlueprinterError, "View '#{view_name}' is not defined"
       end
-      prepared_object = select_columns(object, view_name: view_name)
+      prepared_object = Optimizer.new(object)
```

Comment:
This makes more sense to me as a factory. I expect an optimizer being returned here.